pkg/controllers/dynakube/deploymentmetadata/reconciler.go

```diff
@@ -64,21 +64,23 @@ func (r *Reconciler) addOperatorVersionInfo(configMapData map[string]string) {
 }
 
 func (r *Reconciler) maintainMetadataConfigMap(ctx context.Context, configMapData map[string]string) error {
-	configMapQuery := configmap.NewQuery(ctx, r.client, r.apiReader, log)
+	configMapQuery := configmap.Query(r.client, r.apiReader, log).WithOwner(&r.dk)
```

Comment:
query.WithOwner is not needed (as in case of ag.createTenantTokenSecret)

Reply:
done